agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/SecondEntryPoint.java

```diff
@@ -164,12 +164,12 @@ public void customize(AutoConfigurationCustomizer autoConfiguration) {
     BytecodeUtilImpl.connectionStringConfiguredAtRuntime =
         configuration.connectionStringConfiguredAtRuntime;
 
-    if (configuration.preview.profiler.enabled) {
-      try {
+    try {
+      if (configuration.preview.profiler.isEnabled()) {
```

Comment:
I think it reads slightly easier with the previous if placement outside of the try catch block